src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3918,6 +3918,46 @@ paths:
                   $ref: '#/components/schemas/StandardComment'
         default:
           $ref: '#/components/responses/defaultResponse'
+
+  /license/stdcomments:
```

Comment:
Oops, the key is already defined at line `3902`. Please merge the `put` with the same to pass the test cases.

Reply:
Eish, sorry I don't know how I messed up on this. Let me know if everything looks great now